File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

public class UberVan  extends Car{
    /* -------------------------------------------------------------------------- */
    /*                                  Variables                                 */
    /* -------------------------------------------------------------------------- */
    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;
    private Integer passenger;

    /* -------------------------------------------------------------------------- */
    /*                                 Constructor                                */
    /* -------------------------------------------------------------------------- */

    public UberVan(String license, Account driver, Map<String, Map<String, Integer>> typeCarAccepted,
            ArrayList<String> seatsMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }

    /* -------------------------------------------------------------------------- */
    /*                         Validar numero de pasajeros                        */
    /* -------------------------------------------------------------------------- */
    @Override
    public void setPassenger(int passenger) {
        // TODO Auto-generated method stub
        if (passenger == 6) {
            this.passenger = passenger;
        } else {
            System.out.println("Necesitas asignar 4 pasajeros");
        }
    }

}

File: PHP/UberVan.php
<?php
include_once('Car.php');
class UberVan extends Car
{
    //Variables
    public $typeCarAccepted;
    public $seatMaterial;


    /* -------------------------------------------------------------------------- */
    /*                                 Constructor                                */
    /* -------------------------------------------------------------------------- */
    public function __construct($license, $driver, $typeCarAccepted, $seatMaterial)
    {
        //Hereda los atributos de Car
        parent::__construct($license, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->$seatMaterial = $seatMaterial;
    }

    /* -------------------------------------------------------------------------- */
    /*                                  SET Y GET                                 */
    /* -------------------------------------------------------------------------- */
    public function setPassenger($passenger)
    {
        //Valida que el numero de pasajeros sean 6
        if ($passenger == 6) {
            $this->passenger = $passenger;
        } else {
            echo "Necesitas asignar 6 pasajeros";
        }
    }

}
